lib/server.d.ts

```diff
@@ -1,4 +1,9 @@
-import { ServerArgs } from './';
+import Hapi from '@hapi/hapi';
+import { Dict, ServerArgs } from './';
+declare type ServerType = Hapi.Server & {
+  db?: void | typeof import('mongoose');
```

Comment:
Would we be able to support other dB typings? For example, MySQL or POSTGRESQL.

Reply:
This is the inferred typing from the call to dbConnect, which is quite strange given that it should return a DbClient. I have done a cast to DbClient. Ran the tests and it worked and should work for other db Typings . Would be pushing a commit in that respect.

Reply:
I think you can reuse the DbType declared somewhere here instead @babaalaja

Reply:
@Ehbraheem Unfortunately I can't find this type **DbType**.  Is this preferred 
`type ServerType = Hapi.Server & { db?: DbClient; schema?: Dict };
`

Reply:
Yeah. That's what I mean

Reply:
Ok thanks @Ehbraheem